caffe2/operators/batch_matmul_op.h

```diff
@@ -255,7 +255,7 @@ class BatchMatMulOp final : public Operator<Context> {
 
       // TODO(T23893772): doing this in a loop is likely going to be slow on GPU
```

Comment:
Does this comment still apply?

Reply:
Actually I'm not sure about that. For the batmatmul op, we will update it later to also support broadcasting